pkg/cstorpoolcluster/v1alpha1/cstorpoolcluster.go

```diff
@@ -18,19 +18,18 @@ package v1beta1
 
 import (
 	apisv1alpha1 "github.com/openebs/maya/pkg/apis/openebs.io/v1alpha1"
-	apisv1beta1 "github.com/openebs/maya/pkg/apis/openebs.io/v1beta1"
 )
 
-// SPC encapsulates StoragePoolClaim api object.
+// SPC encapsulates CStorPoolCluster api object.
```

Comment:
Can we rename this struct to CSPC & similarly other names wherever applicable?

Reply:
Yes. Will fix it.